st2actions/st2actions/runners/ssh/paramiko_ssh.py

```diff
@@ -111,6 +111,11 @@ def __init__(self, hostname, port=22, username=None, password=None,
         self.client.set_missing_host_key_policy(paramiko.AutoAddPolicy())
         self.logger = logging.getLogger(__name__)
         self.sftp = None
+        self.bastion_host = bastion_host
```

Comment:
I didn't use `_bastion_host` as the other properties on `ParamikoSSHClient` aren't prefixed.